### ISSUE TITLE
[Bug] - Typo in Docker systemd service file

### ISSUE THREAD

**Issue by kivengh** [amazonlinux/amazon-linux-2023#522]
**Describe the bug**
There is a typo in systemd docker service file, line 20. It's written ``` LimitNOFILE=inifity``` instead of  ```LimitNOFILE=infinity```.

You can see this error in systemd journal : 

```
Oct 20 11:08:39 myhost systemd[1]: /usr/lib/systemd/system/docker.service:20: Failed to parse resource value, ignoring: inifity
Oct 20 11:08:52 myhost systemd[1]: /usr/lib/systemd/system/docker.service:20: Failed to parse resource value, ignoring: inifity
```

**To Reproduce**
1. Install docker
2. Start docker

**Expected behavior**
No more error messages.

**Comment by kivengh** [comment]
I did the same. It works as expected.

**Comment by mwebber** [comment]
Line 20 with the error looks like this:
`LimitNOFILE=inifity`
But I see the same parameter repeated on line 37, this time with the correct spelling:
`LimitNOFILE=infinity`

So the correct fix is presumably to delete, rather than edit, line 20.

**Comment by nmeyerhans** [comment]
This will be fixed in an upcoming release. Thanks for reporting it.

**Comment by nmeyerhans** [comment]
This is fixed as of docker 24.0.5-1.amzn2023.0.3, included in the AL2023.3 release just published.
